clients/ui/frontend/src/__tests__/cypress/cypress/pages/components/Notification.ts

```diff
@@ -0,0 +1,7 @@
+export class Notification {
```

Comment:
Maybe we should rename `ToastNotification`?

Reply:
Yes, we are not using Notification, we might even refactor this in the future if this is not part of the project but reusable code, we'll see

Reply:
At the moment, just rename it and we can call it a day